models/object_detection/model_templates/horizontal-text-detection/readme.md

```diff
@@ -22,165 +27,99 @@ Activate virtual environment:
-it's necessary to download at least one dataset from following resources.
-*  [ICDAR2013 (Focused Scene Text)](https://rrc.cvc.uab.es/?ch=2) - test part is used to get quality metric.
-*  [ICDAR2015 (Incidental Scene Text)](https://rrc.cvc.uab.es/?ch=4)
-*  [ICDAR2017 (MLT)](https://rrc.cvc.uab.es/?ch=8)
-*  [ICDAR2019 (MLT)](https://rrc.cvc.uab.es/?ch=15)
-*  [ICDAR2019 (ART)](https://rrc.cvc.uab.es/?ch=14)
-*  [MSRA-TD500](http://www.iapr-tc11.org/mediawiki/index.php/MSRA_Text_Detection_500_Database_(MSRA-TD500))
-*  [COCO-Text](https://bgshih.github.io/cocotext/)
+### 3. Try a pre-trained model
 
-### 4. Convert datasets
-
-Extract downloaded datasets in `${DATA_DIR}/text-dataset` folder.
+#### a. Change current directory to directory where the model template has been instantiated
 
 ```bash
-export DATA_DIR=${WORK_DIR}/data
+cd ${WORK_DIR}
 ```
+#### b. Export pre-trained PyTorch\* model to the OpenVINO™ format
 
-Convert it to format that is used internally and split to the train and test part.
+To convert PyTorch\* model to the OpenVINO™ IR format run the `export.py` script:
 
-* Training annotation
-```bash
-python ./model_templates/horizontal-text-detection/tools/create_dataset.py \
-    --config ./model_templates/horizontal-text-detection/tools/datasets/dataset_train.json \
-    --output ${DATA_DIR}/text-dataset/IC13TRAIN_IC15_IC17_IC19_MSRATD500_COCOTEXT.json \
-    --root ${DATA_DIR}/text-dataset/
-export TRAIN_ANN_FILE=${DATA_DIR}/text-dataset/IC13TRAIN_IC15_IC17_IC19_MSRATD500_COCOTEXT.json
-export TRAIN_IMG_ROOT=${DATA_DIR}/text-dataset
-```
-* Testing annotation
 ```bash
-python ./model_templates/horizontal-text-detection/tools/create_dataset.py \
-    --config ./model_templates/horizontal-text-detection/tools/datasets/dataset_test.json \
-    --output ${DATA_DIR}/text-dataset/IC13TEST.json \
-    --root ${DATA_DIR}/text-dataset/
-export VAL_ANN_FILE=${DATA_DIR}/text-dataset/IC13TEST.json
-export VAL_IMG_ROOT=${DATA_DIR}/text-dataset
+python export.py \
+   --load-weights ${SNAPSHOT} \
+   --save-model-to export
 ```
 
-Examples of json file for train and test dataset configuration can be found in `horizontal-text-detection/datasets`.
-So, if you would like not to use all datasets above, please change its content.
+This produces model `model.xml` and weights `model.bin` in single-precision floating-point format
+(FP32). The obtained model expects **normalized image** in planar BGR format.
 
-The structure of the folder with datasets:
-```
-${DATA_DIR}/text-dataset
-    ├── coco-text
-    ├── icdar2013
-    ├── icdar2015
-    ├── icdar2017
-    ├── icdar2019_art
-    ├── icdar2019_mlt
-    ├── MSRA-TD500
-    ├── IC13TRAIN_IC15_IC17_IC19_MSRATD500_COCOTEXT.json
-    └── IC13TEST.json
-```
+#### c. Run demo with exported model
 
-### 5. Change current directory to directory where the model template has been instantiated.
+You need to pass a path to `model.xml` file and video device node (e.g. /dev/video0) of your web cam.
```

Comment:
Maybe it is worth defining that -i may contain the path to image/video instead of web cam device that can be used to check demo too.

Reply:
mentioned